components/footer.js

```diff
@@ -65,10 +65,16 @@ export default function Footer() {
               </Link>
             </h1>
             <p>
-              The vis.gl framework suite is an open governance collaboration under the auspices of the
-              OpenJS Foundation.
+              Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and Vis.gl contributors. 
```

Comment:
Nit: We usually spell deck.gl, vis.gl etc using lower case
```suggestion
              Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and vis.gl contributors. 
```